exec/vector/src/main/codegen/templates/FixedValueVectors.java

```diff
@@ -298,6 +298,11 @@ public int getPayloadByteCount(int valueCount) {
     return valueCount * ${type.width};
   }
 
+  @Override
+  public int getValueWidth() {
```

Comment:
If we are using stdSize and get the value from TypeHelper, we don't need all value vectors to have this new function.

Reply:
TypeHelper doesn't return the correct size for Nullable columns. For example NullableIntVector has a width of 5 instead of 4.